Symfony/CS/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+/**
+ * Test that parses and runs the fixture *.test files found in
+ * /Fixtures/Integration.
+ *
+ * @author SpacePossum
+ */
+class IntegrationTest extends AbstractIntegrationTest
```

Comment:
Very well.
Then I suggest to merge it as-is, but having both class `@internal`, then, at 2.x the base class and fixture format will need to be refactred - and then we made base class `@api`.

After #1383 we should be very close to 2.x